deploy/cdk/lib/image-processing/images-stack.ts

```diff
@@ -104,17 +95,19 @@ export class ImagesStack extends cdk.Stack {
       ],
     }))
 
-    const taskDef = new ecs.Ec2TaskDefinition(this, "TaskDefinition", { taskRole })
-    const logging = new ecs.AwsLogDriver({ streamPrefix: 'marbleimg' })
-
     if(!fs.existsSync(props.dockerfilePath)) {
       this.node.addError(`Cannot deploy this stack. Asset path not found ${props.dockerfilePath}`)
       return
     }
+
+    const taskDef = new ecs.FargateTaskDefinition(this, 'TaskDef', {
+      taskRole: taskRole,
+      memoryLimitMiB: 4096,
+      cpu: 2048,
+    })
     taskDef.addContainer("AppContainer", {
       image: ecs.ContainerImage.fromAsset(props.dockerfilePath),
-      memoryReservationMiB: 256,
-      logging,
+      logging: new ecs.AwsLogDriver({ streamPrefix: 'marbleimg' }),
```

Comment:
I think you can make this put logs in the shared log group with something like:
```suggestion
      logging: ecs.AwsLogDriver.awsLogs({
        logGroup: props.foundationStack.logGroup,
        streamPrefix: `${this.stackName}-AppContainer`,
      })
```
You'll def want to test that.

Reply:
ok I gave it a shot and tested it out. looks good to me.